### PR TITLE
Update website language

### PR DESCRIPTION
**Pull request by asettle**
First commit of many updating the website's language around Harvester

### CODE REVIEW

**Comment by asettle** [comment]
Follow-ups: https://github.com/harvester/harvesterhci.io/pull/53/

File: docusaurus.config.js [hunk 1]
```diff
@@ -223,7 +223,7 @@ const config = {
   customFields: {
     title: "Harvester - Open-source hyperconverged infrastructure",
     description:
-      "An open-source hyperconverged infrastructure (HCI) software for a cloud-native world",
+      "An open-source hyperconverged infrastructure software for a cloud-native world",
```
Comment (jillian-maroket) [inline docusaurus.config.js:226]:
```suggestion
      "The open-source hyperconverged infrastructure solution for a cloud-native world",
```

**Review by jillian-maroket — COMMENTED** [review]
Per today's discussion, we will use "solution" exclusively for HCI and Harvester. I suggest that we use "the" (instead of "an") to make a stronger statement.

**Comment by asettle** [comment]
Thanks @jillian-maroket ready and no longer in WIP! The corresponding web update is ready to go: https://github.com/harvester/harvesterhci.io/pull/55

**Review by jillian-maroket — APPROVED** [review]

**Review by markhillgit — APPROVED** [review]
lgtm thanks

**Pull request merged**